region/brinstar/blue/Blue Brinstar Boulder Room.json

```diff
@@ -564,25 +589,6 @@
         }
       }
     },
-    {
-      "id": 31,
-      "link": [2, 2],
-      "name": "Prepared Entry",
```

Comment:
I'm curious about the reasoning for removing this strat. Is it just because the prepared entry is harder than getting out of the water, so it doesn't have much logical purpose?

If it were kept I guess it should have a `{"doorUnlockedAtNode": 2}` requirement. I see some value in maybe keeping it, as it seems like it may still be a potentially useful strat for players?

Reply:
You're probably right that it doesn't have much benefit at the moment. If we cared about timings, it might matter because it can be faster than falling in and jumping back out. The Electric Death one would be more niche since the only use for a prepared entry there would be to reset the room more quickly, compared to Blue Boulder where it can help you get across the room faster. I'm fine with it either way though 🤷‍♂️